Making Sense Project API/Controllers/RentalsController.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Making_Sense_Project_API.Model.Class;
+using Making_Sense_Project_API.Model.Repository;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Making_Sense_Project_API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class RentalsController : ControllerBase
+    {
+        private readonly IRentalsCRUD<Rentals> _rentalsCRUD;
+        private readonly ICarCRUD<Car> _carCRUD;
+        private readonly ICustomerCRUD<Customer> _customerCRUD;
+
+        public RentalsController(IRentalsCRUD<Rentals> rentalsCRUD,
+            ICarCRUD<Car> carCRUD, ICustomerCRUD<Customer> customerCRUD)
+        {
+            _rentalsCRUD = rentalsCRUD;
+            _carCRUD = carCRUD;
+            _customerCRUD = customerCRUD;
+        }
+
+        [HttpPost("Create")]
+        public IActionResult Create(Rentals rentals)
+        {
+            if (_carCRUD.GetById(rentals.IdCar) == null)
+            {
+                return BadRequest($"No hay auto registrado con id {rentals.IdCar}");
```

Comment:
Si queres podes utilizar `nameof` en el mensaje. Es una buena práctica
```suggestion
                return BadRequest($"No hay auto registrado con {nameof(rentals.IdCar)} {rentals.IdCar}");
```